implementing-nosto/advanced-implementation/accessing-state-with-selectors.md

```diff
@@ -1,25 +1,8 @@
 # Accessing state with selectors
 
-`useAppState` hook provides you full access to the entire application state.
+Deprecated `useAppState` hook provides you full access to the entire application state.
 
-{% code title="State example" %}
-```jsx
-import { useAppState } from '@nosto/preact'
-
-export default () => {
-    const state = useAppState()
-	...
-}
-```
-{% endcode %}
-
-While simple, this approach has a number of disadvantages. Namely, any change to the state will trigger a re-render on every component that uses the state; and it may be complicated to access nested data. For better performance and usability, consider the `useAppStateSelector` hook described in the next section
-
-{% code title="Example" %}
-```jsx
-const { response: { products }} = useAppState()
-```
-{% endcode %}
+While simple, this hook has a number of disadvantages. Namely, any change to the state will trigger a re-render on every component that uses the state; and it may be complicated to access nested data. For better performance and usability, consider the `useAppStateSelector` hook described in the next section
```

Comment:
use stronger language:
instead of "consider using ..." use something like "should be used instead"